packages/wonder-blocks-dropdown/components/select-opener.js

```diff
@@ -69,8 +73,14 @@ export default class SelectOpener extends React.Component<SelectOpenerProps> {
 
     static contextTypes = {router: PropTypes.any};
 
+    handleClick = (e: SyntheticEvent<>) => {
+        const {open} = this.props;
+        // TODO: also detect enter click events
```

Comment:
Same comment here, too.

Reply:
Removing comment because we don't want it to detect enters.